ffmpeg/lpms_ffmpeg.c

```diff
@@ -797,13 +797,15 @@ int lpms_transcode(input_params *inp, output_params *params, int nb_outputs)
     }
     ret = open_output(octx, &ictx);
     if (ret < 0) main_err("transcoder: Unable to open output\n");
+    area += params[i].h * params[i].w;
   }
 
   av_init_packet(&ipkt);
   dframe = av_frame_alloc();
   if (!dframe) main_err("transcoder: Unable to allocate frame\n");
 
   while (1) {
+    frames++;
```

Comment:
As far as I can tell (without actually running things and checking), this only counts the number of input frames. A single input frame may produce multiple output frames. Also, there needs to be a check for audio and video; both are carried in AVFrames, but we're only tracking video pixels right now.

Reply:
You're right, but we're currently only counting video frames.  Audio can be handled in a separate pull request - this structure was created to be purposefully extensible.

Reply:
> Audio can be handled in a separate pull request - this structure was created to be purposefully extensible.

Audio frames are still processed within this flow, so it will currently count audio frames as video. Moreover, counting input frames is incorrect, as there isn't a 1-1 correspondence with the number of input frames and number of output frames.

Unit tests are a good idea to double check the numbers that are being produced here.